src/components/ArticleList.jsx

```diff
@@ -0,0 +1,38 @@
+import axios from "axios";
+import { useEffect, useState } from "react";
+import ArticleCard from "./ArticleCard";
+
+export default function ArticleList() {
+    const [articleList, setArticleList] = useState([]);
+    const [isLoading, setIsLoading] = useState(true);
+
+    useEffect(() => {
+        axios
+            .get("https://nc-news-u90o.onrender.com/api/articles/")
+            .then((response) => {
+                return response.data;
+            })
+            .then((data) => {
+                setArticleList(data.articleList);
+                setIsLoading(false);
+            });
+    }, []);
```

Comment:
Please do your axios requests in another file and import the function. Keeps things clean :)